banjax-config.yaml

```diff
@@ -31,6 +31,8 @@ password_protected_path_exceptions:
 password_hashes:
 password_hash_roaming:
   sub.example.com: example.com
+password_persite_cookie_ttl_seconds:
+  example.com: 8787
```

Comment:
Probably using `3600` here (as you did in `fixtures/banjax-config-test.yaml`) will be more explicit for new users. My mind automatically interpreted`8787` as a port number  :smile_cat: